roles/rollback/tasks/main.yml

```diff
@@ -17,6 +17,6 @@
 
 - name: Link 'current' directory to target release
   file:
-    path: "{{ project_root }}/current"
+    path: "{{ project_root }}/{{ project_current_path }}"
```

Comment:
I don't think `project_current_path` exists in rollback. You'd have to define it in its defaults as well.